src/runtime/methodbinder.cs

```diff
@@ -711,12 +711,8 @@ static Type TryComputeClrArgumentType(Type parameterType, IntPtr argument, bool
             else if (positionalArgumentCount < parameters.Length && (!paramsArray || positionalArgumentCount == parameters.Length - 1))
             {
                 match = true;
-                // operator methods will have 2 CLR args but only one Python arg,
-                // since Python operator methods are bound
-                if (isOperator && positionalArgumentCount == parameters.Length - 1)
+                if (positionalArgumentCount == parameters.Length - 1)
```

Comment:
Since isOperator will always be false, the correct change is to remove this branch entirely.

Reply:
Thanks for catching this, I completely forgot that the premature exit was new...